@here/harp-mapview/lib/geometry/TileGeometryManager.ts

```diff
@@ -98,6 +100,15 @@ export interface TileGeometryManager {
      * @returns {GeometryKindSet}
      */
     getAvailableKinds(tiles: Tile[]): GeometryKindSet;
+
+    /**
+     * Sets a callback that will be called for every updated tile on [[updateTiles]].
+     *
+     * @param {TileUpdateCallback} callback The callback that will be called after a tile has been
+     * updated, passing the updated tile as argument. If `undefined`, a previously set callback will
+     * be cleared.
+     */
+    setTileUpdateCallback(callback?: TileUpdateCallback): void;
```

Comment:
It's call "update" because that's the verb the methods TileGeometryManager.updateTiles() and TileGeometryLoader.update() use. 

>  because the tile is loaded from disk
TileGeometryLoader takes a decoded tile, creates objects from its geometry and adds them to the tile. I don't see it loads anything from disk.

Maybe you thought this code was related to the **TileLoader** ?

Reply:
Now I understand, I thought this was called when the tile was finished loading, but now it makes sense.

Reply:
Yes `TileGeometryLoader` takes a decoder tile, but clearing the cache means that the Tile must be re-requested, i.e. the the `TileLoader` is recreated, which will pull the data from the `DataProvider`, hence reloading from disk (which is however likely incorrect, sorry for the potential confusion there, i guess it is in memory).